### ISSUE TITLE
Note that the development build is not optimized. To create a production build, use npm run build.

### ISSUE THREAD

**Issue by kaqiinono** [dvajs/dva-example-user-dashboard#32]
![image](https://user-images.githubusercontent.com/11257690/29078410-e61e0090-7c8c-11e7-8cfe-99141dac6e50.png)

the page is empty

**Comment by sorrycc** [comment]
Have you enabled a proxy?

**Comment by princecharming519** [comment]
How can I enable proxy?

**Comment by ShivaShankerReddy** [comment]
@sorrycc i enabled proxy and go that note inside terminal.

**Comment by blockofcodes** [comment]
Change the URL to http://localhost:3000/  << this is the default set URL when you install react

**Comment by Navdeep-bains** [comment]
after create your react-app and before start your server you have to run this command npm run build after this command start you server i hope this will be great for you

**Comment by emadigit** [comment]
> after create your react-app and before start your server you have to run this command npm run build after this command start you server i hope this will be great for you

hello, I get the same error, could u plz explain more that when I should type npm run build?
tnx

**Comment by avijadu** [comment]
> after create your react-app and before start your server you have to run this command npm run build after this command start you server i hope this will be great for you

I tried it but their is no solution. I also use localhost:3000

**Comment by tahiriabdo131** [comment]
you are forget to export an component

**Comment by UtmostCreator** [comment]
> you are forget to export an component

Yeah, I have exactly this problem. in `babel.config.js` file, but in your case it could be any other file.
Before:
```
// ./babel.config.js
module.exports = {
  presets: ["@vue/cli-plugin-babel/preset"],
};
```
After:
```
// ./babel.config.js
export default module.exports = {
  presets: ["@vue/cli-plugin-babel/preset"],
};
```

However, if I resetar the client, then I have the following error:
```
 ERROR  SyntaxError: DISK:\path\to\my\project\client\babel.config.js: Error while loading config - Unexpected token 'export'
DISK:\path\to\my\project\client\babel.config.js:1
export default module.exports = {
^^^^^^
```

**Comment by abdalshafiealmajdoup** [comment]
install nvm ,
node -v, please select node stable v16.16.0,
nvm install v16.16.0,
nvm now use v16.16.0,
sudo rm -r  package-lock.json,
sudo rm -r node_modules,
sudo rm -r dist,
npm install --save,
npm run build,
npm run serve -- --port 8080
...... Done......

**Comment by coderbekhub** [comment]
> after create your react-app and before start your server you have to run this command npm run build after this command start you server i hope this will be great for you
i did npm run build and i get error in terminal saying failed compiled help me
